index.html

```diff
@@ -15,6 +15,7 @@ <h2 class="name">{{name}}</h2>
           <h4>{{date}}<h4>
           <a href={{link}}>Here is a link!</a>
           <img src="{{img}}" alt="">
+          <h4>Fucks Given: {{fucksGiven}}</h4>
```

Comment:
Oh my. I really like this. How I usually judge things is by how many fucks are given :)